templates/parachain/runtime/src/genesis_config_presets.rs

```diff
@@ -7,6 +7,13 @@ use alloc::{vec, vec::Vec};
 
 use polkadot_sdk::{staging_xcm as xcm, *};
 
+#[doc::include_str!("../../README.md")]
+#[cfg(feature = "generate-readme")]
+
+docify::compile_markdown!("../..","../..");
```

Comment:
I think this should be:
```suggestion
docify::compile_markdown!("../../README.docify.md","../../README.md");
```